osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/CoupleMbusDeviceByChannelCommandExecutorTest.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2021 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.domain.commands.mbus;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.openmuc.jdlms.GetResult;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.entities.DlmsDevice;
+import org.opensmartgridplatform.adapter.protocol.dlms.domain.factories.DlmsConnectionManager;
+import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.ChannelElementValuesDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.CoupleMbusDeviceByChannelRequestDataDto;
+import org.opensmartgridplatform.dto.valueobjects.smartmetering.CoupleMbusDeviceByChannelResponseDto;
+
+@ExtendWith(MockitoExtension.class)
+public class CoupleMbusDeviceByChannelCommandExecutorTest {
+
+    @Mock
+    private DeviceChannelsHelper deviceChannelsHelper;
+
+    @Mock
+    DlmsConnectionManager conn;
+
+    @Mock
+    DlmsDevice device;
+
+    @Mock
+    CoupleMbusDeviceByChannelRequestDataDto coupleMbusDeviceByChannelRequestDataDto;
```

Comment:
Making all Mock fields private would be more consistent. (All command executor tests that were added.)

Reply:
Made all Mock fields private in classes
- CoupleMbusDeviceByChannelCommandExecutorTest
- CoupleMbusDeviceCommandExecutorTest
- DeCoupleMBusDeviceCommandExecutorTest